library/Engine.nucleus.js

```diff
@@ -260,6 +260,8 @@ class NucleusEngine {
         }, nucleusTagsByName);
       });
 
+    await this.$datastore.removeItemByName(ACTION_CONFIGURATION_BY_ACTION_NAME_TABLE_NAME);
```

Comment:
Can you also also remove the other configuration hash, extendable actions and resource structures?

Reply:
Sure, on it.

Reply:
Would it be a good idea to wrap them into methods like `removeAllActionConfiguration` or something? Easier to stub.